src/lib/formats/cp68_dsk.h

```diff
@@ -0,0 +1,48 @@
+// license:BSD-3-Clause
+// copyright-holders:Barry Rodewald
+/*
+ * cp68_dsk.h
+ *
+ *  Created on: 24/06/2014
+ */
+#ifndef MAME_FORMATS_CP68_DSK_H
+#define MAME_FORMATS_CP68_DSK_H
+
+#pragma once
+
+#include "flopimg.h"
+#include "wd177x_dsk.h"
+
+class cp68_format : public wd177x_format
+{
+public:
+	cp68_format();
+
+	virtual const char *name() const override;
+	virtual const char *description() const override;
+	virtual const char *extensions() const override;
+	virtual int identify(util::random_read &io, uint32_t form_factor, const std::vector<uint32_t> &variants) const override;
+	virtual int find_size(util::random_read &io, uint32_t form_factor, const std::vector<uint32_t> &variants) const override;
+	virtual const wd177x_format::format &get_track_format(const format &f, int head, int track) const override;
+
+private:
+	struct sysinfo_sector_cp68
+	{
+		uint8_t unused1[122]{};
+		uint8_t link_start_track = 0;
+		uint8_t link_start_sector = 0;
+		uint8_t link_end_track = 0;
+		uint8_t link_end_sector = 0;
+		uint8_t fc_start_track = 0;
+		uint8_t fc_start_sector = 0;
+	};
+
+	static const format formats[];
+	static const format formats_head1[];
+	static const format formats_track0[];
+	static const format formats_head1_track0[];
```

Comment:
Do these actually need to be class members?  If you throw them in an anonymous namespace in the .cpp file, it reduces the fragility (less chance of unnecessary downstream recompiles).